### PR TITLE
Enable github actions for linting and tests

### PR DESCRIPTION
**Pull request by arielsvn**
## Issue Number

#26 

## Purpose/Implementation Notes

Testing github actions to run tests

## Types of changes

- New feature (non-breaking change which adds functionality)

### CODE REVIEW

**Comment by cgreene** [comment]
BTW not sure if you've seen some of the recipes on their marketplace but you can even get the linter to markup things on github if you use that https://github.com/marketplace/actions/lint-action

File: .github/workflows/push.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+      - uses: pre-commit/action@v1.0.1
+
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Build the stack
+        run: docker-compose build
+      - name: Test
+        run: docker-compose run --rm web bash -c "flake8 . && python wait_for_postgres.py && ./manage.py test"
```
Comment (cgreene) [inline .github/workflows/push.yml]: My impression is that github actions might benefit from having a separate workflow for something like linting and testing. Your linting workflow could be largely derived by the one from the marketplace that I linked. I feel like that's the main value - you can break up large CI processes into small tasks that run on specific triggers. Some might share triggers.

**Comment by arielsvn** [comment]
That looks way better, I'm going to try to install that one instead! 🔥

**Comment by arielsvn** [comment]
Enabled [lint-action](https://github.com/marketplace/actions/lint-action) instead, it's way better! Great recommendation @cgreene 

This should be ready for another review.

**Pull request merged**